### ISSUE TITLE
Multiple button press combo not working

### ISSUE THREAD

**Issue by emdeex** [workinghard/jslisten#20]
Hi, Merry Christmas !  

Thanks for all your help with this!  

Tonight, I'm trying to get the multiple button press logic working.

It doesn't seem to be detecting the multiple button press.  I'm trying to debug it... 

I'm trying 
# sudo /bin/jslisten --loglevel debug

 /var/log/syslog says:

Dec 25 20:48:46 raspberrypi jslisten[2155]: Using device path: /js
Dec 25 20:48:46 raspberrypi jslisten[2155]: Using button mode: plain
Dec 25 20:48:46 raspberrypi jslisten[2155]: Listen to joystick inputs ...
Dec 25 20:48:46 raspberrypi jslisten[2155]: taking user config
Dec 25 20:48:46 raspberrypi jslisten[2155]: reading config /etc/jslisten.cfg
Dec 25 20:48:46 raspberrypi jslisten[2155]: Filename: "/home/pi/combo1.sh
Dec 25 20:48:46 raspberrypi jslisten[2155]: button1: 0
Dec 25 20:48:46 raspberrypi jslisten[2155]: button2: 1

But pressing button 0 and 1 only triggers each button separately... not the combo1.sh

Any thoughts?

**Comment by workinghard** [comment]
Looks to me like you've defined same buttons for different actions. The definition of single button 0 and button 1 gets a precedence over the combination of 0+1. Remove the single definitions and the combination should trigger.

**Comment by emdeex** [comment]
Ah...I see...

Is there a way I can have the single button presses work as well as the combos?  That would be awesome ...is there an area of code I could modify and recompile?

**Comment by workinghard** [comment]
Right now it’s not possible. First a grace time need to be implemented where you have to analyze what’s happening in this time frame. It kind of goes along with #16 feature request.

**Comment by emdeex** [comment]
As a hack...would it be possible to run two processes of jslisten with two
separate configs?

**Comment by workinghard** [comment]
@emdeex Not really. Only if want to trigger both at the same time. Play1, Play2 will be triggered and combination as well.

**Comment by emdeex** [comment]
Thanks that makes sense.

I’ll have a look at the code and see if I can work it out...

**Comment by cgmckeever** [comment]
This may be a similar question (or the same and Im just a bit slow), could two separate instances be  run each listening to a different input? I should try this out ..

**Comment by workinghard** [comment]
> could two separate instances be run each listening to a different input? I should try this out ..
Should work. No resources are being locked by this program.
